Flexible Learning/units_manage_addProcess.php

```diff
@@ -75,20 +84,45 @@
         $file = $_FILES['file'] ?? null;
 
         // Upload the file, return the /uploads relative path
+<<<<<<< Updated upstream
         $data['logo'] = $fileUploader->uploadFromPost($file, $name);
+=======
+        $data['logo'] = $fileUploader->uploadFromPost($file, $name, $data['name']);
```

Comment:
Having fixed this here, are you possibly seeing the same error in editProcess? If so, worth fixing there.

Reply:
Also, I would test this change by creating a new unit, and making sure that the name of the image (as it appears in the database row for the unit) contains the name of the unit.